### DISCUSSION TITLE
CHANGE OF STORAGE LOCATION

### DISCUSSION THREAD

**Discussion by drakub** [YetiForceCompany/YetiForceCRM#17331] [How to]
I am currently finding it cumbersome to store my files in the same directory as the application. I am opting to use a Networ Attached Storage (NAS) to point my files. I have attached the disk, created the folder and pointed uploads there. This works perfectly when uploading the documents. But when I intend to download, the files only open a blank screen.

**Comment by blu-IT** [comment]
This could depend an the filesystem rights / access rights / permissions. I also noticed this problem on local storage, when the permissions where not set propperly to the storage location.
